Assets/Modules/Movement/Scripts/PlayerMovementController.cs

```diff
@@ -66,6 +77,7 @@ private void OnAim(InputValue value)
     private void OnJump(InputValue value)
     {
         rb.isKinematic = false;
```

Comment:
Instead of disabling the PlayerMovementController, you can introduce a new method inside PlayerMovementController called `DisableInput` and then there you should do the following:
First create a field so that you can reference the player input:
![image](https://github.com/SwiffterAnim/Bounce-Escape/assets/47353647/39070964-337e-4438-bb14-86d4f1d3f819)
Then create the disable input method:
![image](https://github.com/SwiffterAnim/Bounce-Escape/assets/47353647/d55ae68f-07df-46a5-84bb-916f054a7806)
Then instead of disabling the script, call this method:
![image](https://github.com/SwiffterAnim/Bounce-Escape/assets/47353647/21cb7a5f-d62c-4c30-9419-332cac35a3f0)

Reply:
Ok this kinda worked, so that made that I can't control the character anymore, but the character is still attracted by the hooks... So I tried, again, to disable the whole playermovementcontroller script, in order for it NOT to be attracted by the hooks, and it still gets attracted somehow, the disabling is working, I see it in the Inspector, but it still gets "hooked". The script that makes it hook is in the PlayerMovementController, the method OnTriggerEnter2D, and it checks if the hook has "hooknessActivated" it hooks the character to the hook.

I still don't understand how it still gets attracted if I'm disabling the whole script haha

Reply:
hmm weird, I just tested it with a joystick and it seems that after dying the player doesn't get hooked.

https://github.com/user-attachments/assets/bfe61820-4514-4d55-bb71-afdab5ef0b8b

Reply:
I should mention that my Unity version is 2022.3.9f1. I highly doubt that it matters but maybe, just "maybe" that could be a bug in your Unity version.

Reply:
Ah yes on the latest version of the build that's already fixed! I fixed it with a boolean "isAlive" to it, and made it work without disabling the script. That was part of my Question 1 on the mentor chat for reference =)x